### PR TITLE
Set up

### LINKED ISSUE #1: DATABASE. Create database
From the Planning story provided, create a PostgreSQL Database fofr storing data.

Ensure integrity of the database.

Implement a cloud service.

Preferably, use Heroku.

Alternatively, use AWS

**Describe alternatives you've considered**

**Additional context**

### LINKED ISSUE #12: VERSIONING. Set up versioning for API
Implement versioning for API.

Implement solution using PHP Lavarel.

### PR DESCRIPTION
**Pull request by Douglasokolaa**
## Description
initial backup set-up
Fixes #1 
Fixes #12 

## Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/api/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+
+const router = express.Router();
+
+router.use('/v1', require('./v1'));
```
Comment (ekrresa) [inline src/api/index.js:5]: We should use ES modules imports

File: src/database/models/skill_categories.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/* jshint indent: 2 */
+
```
Comment (ekrresa) [inline src/database/models/skill_categories.js:2]: What's this for?

**Review by ekrresa — CHANGES_REQUESTED** [review]
I think we should use es modules imports going forward. That's why babel is among the dependencies. Also why are we doing API versioning? The backend is for a single app. I feel it creates unnecessary complexity.

**Comment by Douglasokolaa** [comment]
> I think we should use es modules imports going forward. That's why babel is among the dependencies. Also why are we doing API versioning? The backend is for a single app. I feel it creates unnecessary complexity.

@vector-ok

File: src/api/index.js (hunk 1 above)
Comment (vector-ok) [reply src/api/index.js:5]: Yes. We'll be using ES modules on this project. I'm sure it was an oversight as the commonJS syntax came with the template.

**Comment by vector-ok** [comment]
> I think we should use es modules imports going forward. That's why babel is among the dependencies. Also why are we doing API versioning? The backend is for a single app. I feel it creates unnecessary complexity.

Actually we'll need the versioning. The versioning factor allows companies to walk back on unwanted changes.

**Comment by vector-ok** [comment]
> I think we should use es modules imports going forward. That's why babel is among the dependencies. Also why are we doing API versioning? The backend is for a single app. I feel it creates unnecessary complexity.

Yes. We'll be making use of the ES module on this project. I'm sure it was an oversight as the commonJS syntax came with the template.

**Comment by vector-ok** [comment]
The changes regarding using ES module should be made.

We'll be keeping versioning..

**Pull request closed without merging**